modules/ROOT/pages/customize-layouts.adoc

```diff
@@ -23,14 +23,15 @@ Prerequisites to customize a layout:
 The following example shows how to add a new widget into the layout. +
 The example sections show how to:
 
-* Export the layout from Bonita Portal
+* Export the layout from Bonita Administrator Application
 * Import the layout into the UI Designer
 * Modify the layout
 * Apply the customized layout onto a living application.
 
-== Export a layout from Bonita Portal
+== Export a layout from Bonita Administrator Application
 
-. Open Bonita Portal, and log in with username and password of someone mapped to the Administrator profile.
+. Log in with someone mapped to the Administrator profile.
```

Comment:
Log in with someone mapped to the Administrator profile. => Log in with a users having the Administrator profile.

its not a date to go with someone 👯

Reply:
A lot of people do pair programming, they might log in in pair. Don't discriminate ❤️

Reply:
I added it